harc_game_web/apps/posts/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from django.utils.text import slugify
+from django.utils import timezone
+
+from apps.users.models import HarcgameUser
+from apps.tasks.models import UploadedFile
+
+class Post(models.Model):
+    """
+    Model postu
+    
+    Informacje, które będą zamieszczane dzień po dniu trwania Harcapo, które będa budować fabułę, przypominać o zadaniach itd.
+	Atrybuty:
+		Id postu (generowane automatycznie)
+		Data (dzień i godzina)
+		Tytuł
+		Treść
+		Autor
+		Link do filmiku/obrazu
+    """
+    
+    slug = models.SlugField(max_length=100, unique=True)
+    date_time = models.DateTimeField('Data utworzenia')
+    pub_date_time = models.DateTimeField('Kiedy ma być opublikowany')
+    title = models.CharField(max_length=200)
+    content = models.CharField(max_length=2000)
```

Comment:
maybe we should use `TextField`?

Reply:
https://django.programmingpedia.net/en/knowledge-base/7354588/django-charfield-vs-textfield

I'll remove the chars limit from content